demo/apiExamples.md

```diff
@@ -8,12 +8,19 @@
 |------------------|------------------|-----------|---------|--------------------------------------------------|
 | [disabled](#disabled)       | `disabled`       | `Boolean` |         | If set, disables the combobox.                   |
 | [error](#error)          | `error`          | `Boolean` |         | Sets a persistent error state (e.g. an error state returned from the server). |
+| [noValidate](#noValidate)     | `noValidate`     | `Boolean` |         | If set, disables auto-validation on blur.        |
 | [optionSelected](#optionSelected) | `optionSelected` | `Object`  | null    | Specifies the current selected option.           |
 | [required](#required)       | `required`       | `Boolean` |         | Populates the `required` attribute on the input. Used for client-side validation. |
 | [triggerIcon](#triggerIcon)    | `triggerIcon`    | `Boolean` |         | If set, the `icon` attribute will be applied to the trigger `auro-input` element. |
 | [type](#type)           | `type`           | `String`  |         | Applies the defined value as the type attribute on auro-input. |
 | [value](#value)          | `value`          | `String`  | null    | Value selected for the dropdown menu.            |
 
+## Events
+
+| Event                | Type               |
+|----------------------|--------------------|
+| `auroCombobox-ready` | `CustomEvent<any>` |
```

Comment:
I think this addition to the API docs was a miss from the last updates as I am not seeing this code added here? Please update the reference with an `@fires` identifier and describe the event.